File: python_solutions/sum_square_difference.py
def sum_square_difference(num):
    sum = 0
    sumSquares = 0
    for x in range(1, num + 1):
        sum += x
        sumSquares += (x ** 2)
    
    squareSum = sum ** 2
    return (squareSum - sumSquares)
    
print(sum_square_difference(100))

File: javaScriptSolutions/sumSquareDifference.js
const sumSquareDifference = (num) => {
  let sum = 0;
  let sumSquares = 0;
  for (let i = 0; i <= num; i++) {
    sum += i;
    sumSquares += (i ** 2);
  }
  let squareSum = sum ** 2;
  return (squareSum - sumSquares);
}

console.log(sumSquareDifference(100));
